docsets/Kotlin/docset.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Kotlin",
-    "version": "1.3.70",
-    "archive": "kotlin.tgz",
+    "version": "1.4.21",
+    "archive": "Kotlin.tgz",
     "author": {
         "name": "Alexander Ronald Altman",
```

Comment:
Whoops; here’s a place my name wasn’t replaced.  >.<

(Sorry for not getting around to this myself.)

Reply:
I did not intend to remove it. I guess I'm the maintainer now, but I'm not the author, I used an existing script and some tweaks, so I just left `author` unedited 🙂

Reply:
Oh, I see!  I interpreted “author” differently when I initially took over maintainance.  I guess we’ll have to ask @Kapeli which interpretation was intended?  🤷

Reply:
I have no idea to be honest. I think the current interpretation for most docsets is "current maintainer".